src/gluon/commands/project/CreateGenericProd.ts

```diff
@@ -159,9 +149,9 @@ export class CreateGenericProd extends RecursiveParameterRequestCommand
 
         const tenant = await this.gluonService.tenants.gluonTenantFromTenantId(project.owningTenant);
 
-        await this.ocService.login(QMConfig.subatomic.openshiftNonProd);
+        await this.ocService.login(QMConfig.subatomic.openshiftClouds[this.openShiftCloud].openshiftNonProd);
 
-        const allResources = await this.ocService.exportAllResources(getProjectId(tenant.name, project.name, getHighestPreProdEnvironment().id));
+        const allResources = await this.ocService.exportAllResources(getProjectId(tenant.name, project.name, getHighestPreProdEnvironment(this.openShiftCloud).id));
```

Comment:
Is this right? Isnt it
```suggestion
        const allResources = await this.ocService.exportAllResources(getProjectId(tenant.name, project.name, getHighestPreProdEnvironment(QMConfig.subatomic.openshiftClouds[this.openShiftCloud]).id));
```

Reply:
It was tested and works maybe just reads badly let me refactor a bit thanks